pool/app/filter/repo/repo_utils.ml

```diff
@@ -43,7 +43,7 @@ let add_value_to_params operator value dyn =
   in
   let add c v = Dynparam.add c v dyn in
   match value with
-  | Bool b -> add Caqti_type.bool b
+  | Bool b -> add Caqti_type.string (Utils.Bool.to_string b)
```

Comment:
How about `StdLib.string_of_bool`? ;-)

Reply:
This is what the Utils function calls. 
`let to_string = Bool.to_string`

We can also replace it and call the StdLib function directly. But I thought I will use the Utils since it exists :)